google-cloud-storage/src/main/java/com/google/cloud/storage/BucketInfo.java

```diff
@@ -343,6 +344,7 @@ void populateCondition(Rule.Condition condition) {
      * Sets whether a user accessing the bucket or an object it contains should assume the transit costs
      * related to the access.
      */
+    @GcpLaunchStage.Alpha
     abstract Builder setRequesterPays(Boolean requesterPays);
```

Comment:
If this has already been added to a release, we can't "take back" its status as GA...